.github/workflows/matrix.yml

```
@@ -43,7 +55,7 @@ jobs:
           paths_ignore: '["android/**", "build-data/osx/**", "doc/**", "doxygen_doc/**", "lgtm/**", "msvc-**", "object_creator/**", "tools/**", "utilities/**"]'
   varied_builds:
     needs: skip-duplicates
-    if: ${{ needs.skip-duplicates.outputs.should_skip != 'true' }}
+    if: ${{ needs.skip-duplicates.outputs.should_skip != 'true' || ( matrix.mods == 0 && needs.skip-duplicates.outputs.should_skip_mods != 'true' ) }}
```

First if both say skip, skip.
```suggestion
    if: ${{ needs.skip-duplicates.outputs.should_skip != 'true' && needs.skip-duplicates.outputs.should_skip_mods != 'true' }}
```